app/src/main/java/swati4star/createpdf/activity/MainActivity.java

```diff
@@ -493,20 +523,35 @@ public void setNavigationViewSelection(int id) {
         mNavigationView.setCheckedItem(id);
     }
 
-    private void getRuntimePermissions() {
-        PermissionsUtils.checkRuntimePermissions(this,
-                PERMISSION_REQUEST_CODE,
-                Manifest.permission.WRITE_EXTERNAL_STORAGE,
-                Manifest.permission.READ_EXTERNAL_STORAGE,
-                Manifest.permission.CAMERA);
+    private boolean getRuntimePermissions() {
```

Comment:
there are a number of unnecessary changes. Revert those.

After creating a PR, you should see the "files changed" section. If there is something you didnt change, look into it